test/functional/specs/desktop/C2587.js

```diff
@@ -13,8 +13,6 @@ test.meta({
 test("Regression: Throw error when executing command that doesn't exist", async () => {
   const message = await t.getBrowserConsoleMessages();
   await t
-    .expect(message.warn)
-    .contains(
-      "[alloy] An error during configuration is preventing the boguscommand command from executing."
-    );
+    .expect(message.error)
+    .match(/\[alloy] Resolve these configuration problems:/);
```

Comment:
This doesn't seem to be testing what the spec description is describing. If you're getting a message about configuration problems, you're not even getting past the configure command.